src/v2/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails.tsx

```diff
@@ -1,6 +1,12 @@
 import React, { FC } from "react"
 import { Box } from "@artsy/palette"
+import { SubmissionStepper } from "v2/Apps/Consign/Components/SubmissionStepper"
 
 export const ArtworkDetails: FC = () => {
-  return <Box mt={50}>Artwork Details step</Box>
+  return (
+    <>
+      <SubmissionStepper currentStep="Artwork Details" />
+      <Box mt={50}>Artwork Details step</Box>
```

Comment:
We should never step out of our [spacing scale values](https://palette.artsy.net/atoms/spacing/), and instead round up or down to the closest available option.